spec/domain/queries/find_content_spec.rb

```diff
@@ -156,6 +156,59 @@
     end
   end
 
+  describe 'Filter by all organisations' do
+    let(:filter) do
+      {
+        date_range: 'past-30-days',
+        organisation_id: 'all',
+        document_type: nil
+      }
+    end
+
+    let(:edition1) { create :edition, date: 15.days.ago }
+    let(:edition2) { create :edition, date: 15.days.ago }
+
+    before do
+      create :metric, edition: edition1, upviews: 20, date: 15.days.ago
+      create :metric, edition: edition2, upviews: 10, date: 15.days.ago
+      recalculate_aggregations!
+    end
+
+    it 'returns content from all organisations' do
+      response = described_class.call(filter: filter)
+      expect(response[:results]).to contain_exactly(
+        hash_including(base_path: edition1.base_path),
+        hash_including(base_path: edition2.base_path)
+      )
+    end
+  end
+
+  describe 'Filter by null organisations' do
+    let(:filter) do
+      {
+        date_range: 'past-30-days',
+        organisation_id: 'none',
+        document_type: nil
+      }
+    end
+
+    let(:edition1) { create :edition, date: 15.days.ago, organisation_id: nil }
+
+    before do
+      create :metric, edition: edition1, date: 15.days.ago
+      edition2 = create :edition, date: 15.days.ago, organisation_id: primary_org_id
+      create :metric, edition: edition2, date: 15.days.ago
```

Comment:
This code isn't consistent with code in the previous test - an edition is being created in the `let` block and the `before` block . Again you can also specify facts directly in the edition.